### PR TITLE
Update cluster privileges list

### PR DESCRIPTION
**Pull request by delvedor**
Fixes: #1644

### CODE REVIEW

File: specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts [hunk 1]
```diff
@@ -23,5 +23,6 @@ import { RequestBase } from '@_types/Base'
  * @rest_spec_name security.get_builtin_privileges
  * @since 7.3.0
  * @stability stable
+ * @cluster_privileges manage_enrich
```
Comment (ywangd) [inline specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts:26]: Forgive my ignorance: Not sure why `manage_enrich` is specifically needed here. Is this comment kinda a change history?

File: specification/security/put_role/SecurityPutRoleRequest.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ import { Metadata, Name, Refresh } from '@_types/common'
  * @rest_spec_name security.put_role
  * @since 0.0.0
  * @stability stable
- * @cluster_privileges manage_security
+ * @cluster_privileges manage_enrich, manage_security
```
Comment (ywangd) [inline specification/security/put_role/SecurityPutRoleRequest.ts:37]: Same as above.

**Review by ywangd — APPROVED** [review]
LGTM

The following comments are rather questions due to my lack of knowledge of the project. I appreciate if you could help me understand the changes.

File: specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts (hunk 1 above)
Comment (delvedor) [reply specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts:26]: Happy to answer every question! In the issue you wrote:

> The manage_enrich cluster privilege is missing from:
> Response of the [Get builtin privileges API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-builtin-privileges.html)
> Request of the [Create/Update role API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-role.html) where this privilege should be a candidate for value of the cluster field.

This change adds the `manage_enrich` to the two APIs. If you look at the `schema.json`, now the new privileges are listed there as well.

Comment (sethmlarson) [reply specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts:26]: @delvedor From my reading the enum value was only missing in the response itself, not as a `@cluster_privileges` annotation for this API so adding the value to the enum is sufficient. From the docs it seems that the privilege needed for this API is actually `manage_security`.

Comment (ywangd) [reply specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts:26]: Sorry for any confusion that may have been caused by my original issue. @sethmlarson is right. The cluster privilege requried for calling this API is `manage_security`. The response of this API should return a list of cluster privileges which should include the missing `manage_enrich` privilege. 

Btw: I am assuming the `@cluster_privileges` directive is to specify what cluster privilege is required for calling the API.

File: specification/security/put_role/SecurityPutRoleRequest.ts (hunk 2 above)
Comment (ywangd) [reply specification/security/put_role/SecurityPutRoleRequest.ts:37]: Based on the above, we should remove `manage_enrich` from here as well.

**Pull request merged**